### PR TITLE
Translate ruby 2.4.0-preview2 released (ko)

### PR DESCRIPTION
**Pull request by riseshia**
@ruby/www-ruby-lang-org-i18n-ko
Please Review this! XD

### CODE REVIEW

**Comment by marocchino** [comment]
LGTM.

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+루비 2.4.0-preview2은 루비 2.4.0의 두 번째 프리뷰입니다.
+이 프리뷰는 커뮤니티의 반응을 살펴보고 위해 릴리스되었습니다.
+기능에서 혼란을 느낀다면 부담없이 [피드백](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)을 보내주세요.
+
+## [Fixnum과 Bignum을 Integer로 통합](https://bugs.ruby-lang.org/issues/12005)
+
+[ISO/IEC 30170:2012](http://www.iso.org/iso/iso_catalogue/catalogue_tc/catalogue_detail.htm?csnumber=59579)에서는
+Integer 클래스의 세부에 관해서 기술하고 있지 않아,
+CRuby에서는 2개의 정수 클래스(Fixnum과 Bignum)를 가지고 있습니다.
```
Comment (stomar) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: CRuby -> Ruby

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+## 디버깅
+
+### [Thread#report_on_exception과 Thread.report_on_exception](https://bugs.ruby-lang.org/issues/6647)
+
+루비는 다른 스레드에서 명시적으로 접근하지 않는 이상 스레드에서의 예외를 무시합니다.
+`report_on_exception = true`를 사용하면,
+처리되지 않은 예외 때문에 스레드가 사망하는 경우, 알림을 받을 수 있게 됩니다.
+
+`report_on_exception`의 기본값으로 무엇을 사용하면 좋을지,
+그리고 report-on-GC에 대한 피드백을 보내주세요.
```
Comment (stomar) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: report-on-GC is now explained in `en`, see 030c550514fe3aa61904de2aaa2217a2e94fe63c

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+author: "naruse"
+translator: "shia"
+date: 2016-09-08 18:00:00 +0900
+lang: ko
+---
+
+루비 2.4.0-preview2의 릴리스를 알리게 되어 기쁘게 생각합니다.
+
+루비 2.4.0-preview2은 루비 2.4.0의 두 번째 프리뷰입니다.
+이 프리뷰는 커뮤니티의 반응을 살펴보고 위해 릴리스되었습니다.
```
Comment (yous) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: 살펴보고 위해 → 살펴보기 위해

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+### [Array#max, Array#min](https://bugs.ruby-lang.org/issues/12172)
+
+`[x, y].max`와 `[x, y].min`은 특정 조건에서 임시 배열을 생성하지 않도록
+최적화되었습니다.
+
+### [Regexp#match?](https://bugs.ruby-lang.org/issues/8110)
+
+`Regexp#match?`가 추가되었으며, 이는 후방 참조 객체를 생성하지 않고
+`$~`을 변경하여 객체 할당을 줄이며 정규 표현식을 매칭합니다.
```
Comment (yous) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: > 후방 참조 객체를 생성하지 않고 \`$~\`을 변경하여 객체 할당을 줄이며

How about '후방 참조 객체를 생성하거나 \`$~\`을 변경하지 않음으로써 객체 할당을 줄이며'? See https://github.com/ruby/ruby/commit/f09574c879fe9a4488164a41b453617b4b1d56bb#diff-2c86fd8e421d681ea5365ab4c0624707R3210.

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+그리고 report-on-GC에 대한 피드백을 보내주세요.
+
+### [스레드 교착상태 탐지가 이제 백트레이스와 의존성 정보를 함께 보여주게 됩니다](https://bugs.ruby-lang.org/issues/8214)
+
+루비는 대기 중인 스레드의 교착상태 탐지를 지원합니다만,
+해당 리포트에서는 디버깅을 위한 충분한 정보가 포함되지 않았습니다.
+루비 2.4의 교착상태 탐지는 스레드의 백트레이스와 의존하고 있는 스레드에 대한 정보를 보여주게 됩니다.
+
+루비 2.4.0-preview2로 즐겁게 프로그램을 작성해보세요.
+그리고 여러분의 [느낀 점을 알려주세요](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)!
```
Comment (yous) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: 여러분의 → 여러분이

**Comment by yous** [comment]
Look good to me when other comments are resolved.

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md (hunk 1 above)
Comment (riseshia) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: 👍

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md (hunk 2 above)
Comment (riseshia) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: 👍

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md (hunk 3 above)
Comment (riseshia) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: 👍

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md (hunk 4 above)
Comment (riseshia) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: 👍

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md (hunk 5 above)
Comment (riseshia) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: 👍

**Comment by riseshia** [comment]
Applied all comment :)

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+루비 2.4.0-preview2은 루비 2.4.0의 두 번째 프리뷰입니다.
+이 프리뷰는 커뮤니티의 반응을 살펴보기 위해 릴리스되었습니다.
+기능에서 혼란을 느낀다면 부담없이 [피드백](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)을 보내주세요.
+
+## [Fixnum과 Bignum을 Integer로 통합](https://bugs.ruby-lang.org/issues/12005)
+
+[ISO/IEC 30170:2012](http://www.iso.org/iso/iso_catalogue/catalogue_tc/catalogue_detail.htm?csnumber=59579)에서는
+Integer 클래스의 세부에 관해서 기술하고 있지 않아,
+Ruby에서는 2개의 정수 클래스(Fixnum과 Bignum)를 가지고 있습니다.
```
Comment (yous) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: Ruby → 루비

Comment (riseshia) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: ( ﾟдﾟ) ! Thanks!

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+## 디버깅
+
+### [Thread#report_on_exception과 Thread.report_on_exception](https://bugs.ruby-lang.org/issues/6647)
+
+루비는 다른 스레드에서 명시적으로 접근하지 않는 이상 스레드에서의 예외를 무시합니다.
+`report_on_exception = true`를 사용하면,
+처리되지 않은 예외 때문에 스레드가 사망하는 경우, 알림을 받을 수 있게 됩니다.
+
+`report_on_exception`의 기본값으로 무엇을 사용하면 좋을지,
+그리고 스레드가 조인이 없는 GCed일 때 보고서를 보여주는 report-on-GC에 대한 피드백을 보내주세요.
```
Comment (yous) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: How about '스레드가 조인되지 않은 채 GC 되었을 때'?

Comment (riseshia) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: oh, i thought `GCed` is some keyword OTL.
thanks!

File: ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+## 디버깅
+
+### [Thread#report_on_exception과 Thread.report_on_exception](https://bugs.ruby-lang.org/issues/6647)
+
+루비는 다른 스레드에서 명시적으로 접근하지 않는 이상 스레드에서의 예외를 무시합니다.
+`report_on_exception = true`를 사용하면,
+처리되지 않은 예외 때문에 스레드가 사망하는 경우, 알림을 받을 수 있게 됩니다.
+
+`report_on_exception`의 기본값으로 무엇을 사용하면 좋을지,
+그리고 스레드가 조인되지 않은 채 GC 되었을 때 보고서를 보여주는 report-on-GC에 대한 피드백을 보내주세요.
```
Comment (yous) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: How about just '알려주는' instead of '보고서는 보여주는'?

Comment (riseshia) [inline ko/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md]: 👍

**Comment by riseshia** [comment]
Thanks for a lot reviews! I will merge this 👍

**Comment by stomar** [comment]
:+1:

**Pull request merged**